frontend/src/scenes/App.tsx

```diff
@@ -77,6 +93,8 @@ function _App(): JSX.Element {
         )
     }
 
+    if (!currentOrganization?.id || !currentTeam?.id) return null
```

Comment:
So when is this the case? Do we really want to show a blank page then or do we always redirect the user somewhere?

Reply:
We do redirect, but inside this same `_App` component, but with `useEffect`.